customization/clientSettings.sqf

```diff
@@ -6,4 +6,32 @@ GVAR(StartOnSafe) = true;
 GVAR(StartOnSafe_Lowered) = true;
```

Comment:
Nice to see that we've set a precendent with this "settings" files, should make it easy for mission makers to work out what to tweak to suit their mission :)